templates/pokemon.html

```diff
@@ -49,7 +50,7 @@
       <div class="col-sm-3 col-md-2 sidebar">
         <div class="panel panel-default">
           <div class="panel-body">
-            Level: {{player['level']}}<br> Exp: {{player['experience']}}<br> EXP/Hour: {{player['hourly_exp']|int}} <br> Stardust: {{currency}}<br> Pokemon Count: {{pokemons|length}}<br>
+            Level: {{player['level']}}<br> Exp: {{player['experience']}}<br> EXP/Hour: {{player['hourly_exp']|int}} <br> Stardust: {{currency}}<br> Pokemon Count: {{pokemons|length}}<br> Item Count: {{items|length}}<br>
```

Comment:
I know, but I think it would be cool also to see how many more you can carry if you can't remember - maybe when running multiple bots it can be difficult to differ between them (some of the can have expanded bag sizes - at least some of my accounts does).